css/style.css

```diff
@@ -161,7 +162,7 @@ body{
     }
 
     .main{
-        height: 405vh;
+        height: 350vh;
```

Comment:
A couple of issues here
- Don't use viewport height `vh` to set the height of the main page. You want to set a minimum height but also allow the div expand as the content grows.

```
min-height: 1000px;
```